### ISSUE TITLE
The view and the model should be separated 

### ISSUE THREAD

**Issue by valentiniliescu** [ericgu/AnExerciseInCouplingDesRefWG#6]
The Form class contains both the view and the model
[https://github.com/ericgu/AnExerciseInCouplingDesRefWG/blob/master/StockTracker/Form1.cs](https://github.com/ericgu/AnExerciseInCouplingDesRefWG/blob/master/StockTracker/Form1.cs)

**Comment by bastte** [comment]
Agreed. That's what I brushed on in my issue about data binding (https://github.com/ericgu/AnExerciseInCouplingDesRefWG/issues/4)

**Comment by DanielOlewski** [comment]
This is one possible **solution**. but we were asked to list issues as "problem descriptions". can we reword this one to match the ask?
